pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/generate/PropertyInfo.java

```diff
@@ -21,16 +22,19 @@
 public class PropertyInfo {
 
     public static final Map<String, PropertyInfo> TABLE_ALIAS_MAPPINGS = new HashMap<>(
-            Map.of(DBConstants.DDP_MEDICAL_RECORD_ALIAS, new PropertyInfo(MedicalRecord.class, true),
-                    DBConstants.DDP_TISSUE_ALIAS, new PropertyInfo(Tissue.class, true),
-                    DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS, new PropertyInfo(OncHistoryDetail.class, true),
-                    DBConstants.DDP_PARTICIPANT_DATA_ALIAS, new PropertyInfo(ParticipantData.class, true),
-                    DBConstants.DDP_PARTICIPANT_RECORD_ALIAS, new PropertyInfo(Participant.class, false),
-                    DBConstants.DDP_PARTICIPANT_ALIAS, new PropertyInfo(Participant.class, false),
-                    DBConstants.DDP_ONC_HISTORY_ALIAS, new PropertyInfo(OncHistory.class, false),
-                    DBConstants.SM_ID_TABLE_ALIAS, new PropertyInfo(SmId.class, true),
-                    DBConstants.COHORT_ALIAS, new PropertyInfo(CohortTag.class, true),
-                    DBConstants.DDP_KIT_REQUEST_ALIAS, new PropertyInfo(KitRequestShipping.class, true)));
+            Map.ofEntries(
+                    Map.entry(DBConstants.DDP_MEDICAL_RECORD_ALIAS, new PropertyInfo(MedicalRecord.class, true)),
+                    Map.entry(DBConstants.DDP_TISSUE_ALIAS, new PropertyInfo(Tissue.class, true)),
+                    Map.entry(DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS, new PropertyInfo(OncHistoryDetail.class, true)),
+                    Map.entry(DBConstants.DDP_PARTICIPANT_DATA_ALIAS, new PropertyInfo(ParticipantData.class, true)),
+                    Map.entry(DBConstants.DDP_PARTICIPANT_RECORD_ALIAS, new PropertyInfo(Participant.class, false)),
+                    Map.entry(DBConstants.DDP_PARTICIPANT_ALIAS, new PropertyInfo(Participant.class, false)),
+                    Map.entry(DBConstants.DDP_ONC_HISTORY_ALIAS, new PropertyInfo(OncHistory.class, false)),
+                    Map.entry(DBConstants.SM_ID_TABLE_ALIAS, new PropertyInfo(SmId.class, true)),
+                    Map.entry(DBConstants.COHORT_ALIAS, new PropertyInfo(CohortTag.class, true)),
+                    Map.entry(DBConstants.DDP_KIT_REQUEST_ALIAS, new PropertyInfo(KitRequestShipping.class, true)),
+                    Map.entry(DBConstants.INVITAE_ALIAS, new PropertyInfo(InvitaeReport.class, false))));
```

Comment:
as I see from mapping `InvitaeReport` is type of nested, in this case I believe we need to pass true instead of false here to indicate that it is collection, collection property in this class actually means `nested` if we talk about elastic search.